### ISSUE TITLE
status_effects.csv has zeros in effect thresholds

### ISSUE THREAD

**Issue by palonsoro** [VZGCoders/TOTK-Recipe-Calculator#9]
First of all, thx for working at this!

That said, while trying to have a look and understand how this was implemented, I realized that [status_effects.csv](https://github.com/VZGCoders/TOTK-Recipe-Calculator/blob/main/src/TOTK/CSVs/status_effects.csv) has many effect thresholds at 0, likely as if they were unknown.

I am not an expert on these things (just casual reader about BotW/TotK reverse engineering), but I think I recall some of these were already known in BoTW (e.g. `AttackUp`) and I have recently found [a datasheet](https://github.com/VZGCoders/TOTK-Recipe-Calculator/blob/main/src/TOTK/CSVs/status_effects.csv) which seems to have the complete ones for TotK (BTW, they mention you in the intro sheet, so maybe you already know them).

Would it make sense to update the [status_effects.csv](https://github.com/VZGCoders/TOTK-Recipe-Calculator/blob/main/src/TOTK/CSVs/status_effects.csv)  with the thresholds from the [sheet](https://github.com/VZGCoders/TOTK-Recipe-Calculator/blob/main/src/TOTK/CSVs/status_effects.csv)?

**Comment by valzargaming** [comment]
Thanks for reaching out! Your link to the datasheet seems to link to the status_effects.csv file, I'm assuming that was a mistake? The data dump sheet we've been using is the one Echocolat maintains [here](https://docs.google.com/spreadsheets/d/18pNtDx3z-8CwGJRmlW574xbQ6VphQOkvpZhClpOEVDA/edit#gid=48950788) and have customized for use with the calculator. There's a lot of things that aren't right with it, such as how the Meals are not in order of priority (yet), but we actually have been working closely with them.

**Comment by palonsoro** [comment]
Yep, it was a mistake and the sheet I wanted to link was the Echocolat one: https://docs.google.com/spreadsheets/d/18pNtDx3z-8CwGJRmlW574xbQ6VphQOkvpZhClpOEVDA/edit#gid=1214714862

Ok, so as I was imagining, this is just work in progress and you already work with them. Worth having tried anyway. Thx!

**Comment by valzargaming** [comment]
I'll definitely need to find time to go through and add them. The one I've been maintaining is [here](https://docs.google.com/spreadsheets/d/1Eo2nVcV-UoJKCr5qB8dM8URb5FQwOMtOeUW8oLtKkVA/edit?usp=sharing) but a lot of stuff is either still outdated or we've added stuff from our own findings, removed stuff that isn't relevant to the project, or we're still researching.
